src/components/TexterStats.jsx

```diff
@@ -61,7 +61,7 @@ class TexterStats extends React.Component {
     if (unrepliedCount) {
       bottomLinks.push(
         <span>
-          {unrepliedCount} contacts{" "}
+          {unrepliedCount} contact{unrepliedCount > 1 && "s"}{" "}
```

Comment:
What's up with the space character in the `{" "}` preceding this? Can this instead be:
`contact{unrepliedCount > 1 ? "s ": " "}`

or is that space doing something else?

Reply:
Thanks, `contact{unrepliedCount > 1 ? "s ": " "}` is a better way to add the space character. Updated the PR :)